tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py

```diff
@@ -37,3 +38,25 @@ def test_get_model() -> None:
         model.measurement_aggregate_document.measurement_identifier = ""
 
     assert model == get_model()
+
+
+@pytest.mark.parametrize("output_file", output_files)
```

Comment:
Let's remove the parameterization on this test, we only need one instance of this.

Reply:
Resolved in [5833c79](https://github.com/Benchling-Open-Source/allotropy/pull/72/commits/5833c79bdfb6cabff12553304bb83d5ba3dfdf67)